src/views/product-integrations-landing/components/searchable-integrations-list/index.tsx

```diff
@@ -78,117 +109,22 @@ export default function SearchableIntegrationsList({
+	}, [queryParams.filterQuery, filteredIntegrations.length])
 	useTypingDebounce(searchedEventCallback)
 
-	const {
-		tierOptions,
-		tiersCheckedArray,
-		setTiersCheckedArray,
-		sortedComponents,
-		componentCheckedArray,
-		setComponentCheckedArray,
-		flags,
-		flagsCheckedArray,
-		setFlagsCheckedArray,
-		types,
-		typesCheckedArray,
-		setTypesCheckedArray,
-		atLeastOneFacetSelected,
-	} = useIntegrationsSearchContext()
-
-	// handleClearFilters resets the state of all filters
-	const handleClearFilters = (e) => {
-		resetPage()
-
-		setFilterQuery('')
-
-		setTiersCheckedArray(tiersCheckedArray.map((v, i) => false))
-		setComponentCheckedArray(componentCheckedArray.map((v, i) => false))
-		setFlagsCheckedArray(flagsCheckedArray.map((v, i) => false))
-		setTypesCheckedArray(typesCheckedArray.map((v, i) => false))
-	}
-
-	const makeToggleTierHandler = (index: number, tier: Tier) => () => {
-		// reset page on filter change
-		resetPage()
-
-		// small wrapper to trigger analytics event if given a `true` value
-		const fireTierAnalytics = (value: boolean) => {
-			if (value) {
-				integrationLibraryFilterSelectedEvent({
-					filter_category: 'tier',
-					filter_value: tier,
-				})
-			}
-		}
-
-		const newTierCheckedArray = [...tiersCheckedArray]
-		newTierCheckedArray[index] = !newTierCheckedArray[index]
-		setTiersCheckedArray(newTierCheckedArray)
-
-		fireTierAnalytics(newTierCheckedArray[index])
-	}
-
-	const makeToggleFlagHandler = (i: number, flagName: string) => () => {
-		// reset page on filter change
-		resetPage()
-
-		const newFlags = [...flagsCheckedArray]
-		const isFlagSelectedInNext = !newFlags[i]
-		// When any flag input is checked, track an analytics filtered event
-		if (isFlagSelectedInNext) {
-			integrationLibraryFilterSelectedEvent({
-				filter_category: 'flag',
-				filter_value: flagName,
-			})
-		}
-
-		newFlags[i] = isFlagSelectedInNext
-		setFlagsCheckedArray(newFlags)
-	}
-
-	const makeToggleComponentHandler =
-		(i: number, componentName: string) => () => {
-			// reset page on filter change
-			resetPage()
-
-			const newComponents = [...componentCheckedArray]
-			const isComponentSelectedInNext = !newComponents[i]
-			if (isComponentSelectedInNext) {
-				integrationLibraryFilterSelectedEvent({
-					filter_category: 'component',
-					filter_value: componentName,
-				})
-			}
-			newComponents[i] = isComponentSelectedInNext
-			setComponentCheckedArray(newComponents)
-		}
-
-	const makeToggleTypeHandler = (i: number, typeName: string) => () => {
-		// reset page on filter change
-		resetPage()
-
-		const newTypes = [...typesCheckedArray]
-		const isTypeSelectedInNext = !newTypes[i]
-		if (isTypeSelectedInNext) {
-			integrationLibraryFilterSelectedEvent({
-				filter_category: 'type',
-				filter_value: typeName,
-			})
-		}
-		newTypes[i] = isTypeSelectedInNext
-		setTypesCheckedArray(newTypes)
+	/**
+	 * @TODO replace with actual loading skeleton (currently on different branch)
+	 */
```

Comment:
❓ We probably want to update this before merging? I think returning `null` is a better temporary than the `<h1>`, while the loading skeleton PR is updated as a fast-follow.